fedimintd/src/ui.rs

```diff
@@ -253,6 +256,9 @@ async fn params_page(
     UrlConnection {
         ro_bitcoin_rpc_type,
         ro_bitcoin_rpc_url,
+        bitcoin_network: env::var("FM_BITCOIND_NETWORK").unwrap_or("".to_string()),
```

Comment:
bitcoin vs BITCOIND

I think we should just go with "bitcoin" (no d). It's about general network selector, and is not bitcoind specific.

Reply:
Ah yes, I'll change this.